components/Ripple/src/MDCStatefulRippleView.h

```diff
@@ -14,22 +14,6 @@
 
 #import "MDCRippleView.h"
 
-/**
- Provides the current state of the ripple. The ripple is either in its normal state, or in the
- selected state where the ripple remains spread on the view.
-
- - MDCRippleStateNormal: The ripple isn't currently presented.
- - MDCRippleStateHighlighted: The ripple is activated and shown.
- - MDCRippleStateSelected: The ripple is in the selected state.
- - MDCRippleStateDragged: The ripple is in the dragged state.
- */
-typedef NS_OPTIONS(NSInteger, MDCRippleState) {
-  MDCRippleStateNormal = 0,
-  MDCRippleStateHighlighted = 1 << 0,
-  MDCRippleStateSelected = 1 << 1,
-  MDCRippleStateDragged = 1 << 2,
-};
```

Comment:
This is a breaking change for clients; what would a more gradual migration plan look like for clients?